readtags.c

```diff
@@ -472,7 +472,7 @@ static tagFile *initialize (const char *const filePath, tagFileInfo *const info)
 		result->fields.max = 20;
 		result->fields.list = (tagExtensionField*) calloc (
 			result->fields.max, sizeof (tagExtensionField));
-		result->fp = fopen (filePath, "r");
+		result->fp = fopen (filePath, "rb");
```

Comment:
@b4n, thank you for the comment.
Without this, fsetpos/fgetpos(and fseek/ftell) don't work as I expected. Looks very strange for me.
I will write smaller test program to understand what really happens.